### DISCUSSION TITLE
Testing: The module is a provisional

### DISCUSSION THREAD

**Discussion by ppena-LiveData** [aio-libs/aiohttp#7234] [Q&A]
https://docs.aiohttp.org/en/stable/testing.html says, "The module is a provisional," but there are many different modules (and packages) discussed on that page, so it's not clear what is provisional. Does someone know what exactly is provisional?

**Comment by Dreamsorcerer** [comment]
Well, it refers to aiohttp.test_tools. That module is no longer named that, and the note was added 7 years ago. So, I think we can probably just remove that notice today.
